### ISSUE TITLE
[STORY] add catalogue of parcellation images to T1 processing pipeline

### ISSUE THREAD

**Issue by arkiev** [Australian-Imaging-Service/pipelines#247]
### Metadata

Epic: #17 
Feature: #79 
Feature Release:
Required knowledge: shallow or mid-level or deep

### Description

As a researcher, I want to have a selection of parcellation images to choose from, so that I am less restricted with cortical/sub-cortical structures of interest 

This can be done by converting the parcellation component of https://github.com/bids-apps/MRtrix3_connectome into a format that is compatible with Pydra

### Acceptance Criteria

Given #230 can successfully be executed, if the user requires to use a one of the following parcellation schemes (which are dependant on Free/FastSurfer), then the user can input the parcellation scheme, and the pipeline will generate the corresponding parcellation image(s).

List of parcellation images (first three in order of ease)

- [x] desikan
- [x] destrieux
- [x] hcpmmp1
- [x] yeo17fs
- [x] yeo7fs
- [x] brainnetome246fs

### List of secondary tasks
- [x] provide an option for the user to provide multiple parcellation inputs, and for a given iteration of FastSurfer and 5TTgen, several parcellation images are returned (currently, additional parcellations will require re-running FastSurfer and 5TTgen) 
- [ ] expand the catalogue to include parcellations that are not part of Free/FastSurfer (create separate story for this later)
- [ ] run parcellation image generation in parallel 
- [ ] tidy up working directory automatically (some sort of cleanup). Keep FS output dir.                                   

```[tasklist]
### Bugs to fix
- [ ] update `aparc_2_aseg.py` to include an output field (conversion of freesurfer's `mri_aparc2aseg` did not provide an output field in `aparc_2_aseg.py`
- [x] Yeo7fs and Yeo17fs appear to be mislabelled. Check the LUTs and potentially use the LUT contained in ConnectomeBids Output dir, then re-run
```

**Comment by arkiev** [comment]
secondary task involving multiple parcellation inputs marked as complete, although this has been changed to automatically iterate over all parcellation images available

**Comment by arkiev** [comment]
brainnetome246fs has been built as a standalone pydra script (using FreeSurfer directory as input), but has not been integrated with T1_Processing_pipeline.py. This is because:

1. BN atlas subdivides FS SGM structures, and it is therefore not sensible to use with 5TThsvs
2. FastSurfer does not create Talairach.m3z transform. This can be created (see https://surfer.nmr.mgh.harvard.edu/fswiki/ReconAllTableStableV6.0). This requires $FREESURFER_HOME/average/RB_all_2016-05-10.vc700.gca
